util/strutil/strconv.go

```diff
@@ -78,7 +78,7 @@ func StringToDuration(durationStr string) (duration time.Duration, err error) {
 	case "s":
 		duration *= 1
 	default:
-		panic("Invalid time unit in duration string.")
+		return 0, fmt.Errorf("Invalid time unit in duration string: %q", unit)
```

Comment:
Error strings should not be capitalized (correctly done in line 61).

https://code.google.com/p/go-wiki/wiki/CodeReviewComments#Error_Strings